eZ/Publish/Core/REST/Server/Input/Parser/ViewInput.php

```diff
@@ -36,19 +37,28 @@ public function parse( array $data, ParsingDispatcher $parsingDispatcher )
         $restViewInput = new RestViewInput();
 
         // identifier
-        if ( !array_key_exists( 'identifier', $data ) )
-        {
+        if (!array_key_exists( 'identifier', $data )) {
             throw new Exceptions\Parser( "Missing <identifier> attribute for <ViewInput>." );
         }
-        $restViewInput->identifier  = $data['identifier'];
+        $restViewInput->identifier = $data['identifier'];
 
         // query
-        if ( !array_key_exists( 'Query', $data ) || !is_array( $data['Query'] ) )
-        {
+        if (!array_key_exists( 'Query', $data ) || !is_array( $data['Query'] )) {
```

Comment:
regarding this, isn't phpstorm supposed to keep coding standards per project? whenever i change to a drupal one i need to set ez style again...